README.md

```diff
@@ -15,12 +15,14 @@ QUICKINSTALL:
 
 ```
 # cd espressopp
-# cmake .
-# make
+# cmake -DEXTERNAL_BOOST=OFF -DEXTERNAL_MPI4PY=OFF .

```

Comment:
by the way, on espresso-pp.de we recommend:
```
If cmake . doesn't find all the libraries and include files use ccmake . to manually specify all search paths or to change compiler options.

If you have problems building ESPResSo++ with the system boost- and/or mpi4py- libraries, you can choose to use the ESPResSo++ contributed libraries with the command: (do a make clean before)
# cmake -DEXTERNAL_BOOST=OFF -DEXTERNAL_MPI4PY=OFF -DFFTW3_LIBRARIES=your_lib_path/libfftw3.a . 
# make
```

maybe we should use this form here as well?